cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -79,38 +84,60 @@ class modal(View):  # noqa
 
 
 class ButtonGroupForm(ServicesCatalogView):
-    title = Text('#explorer_title_text')
+    title = Text("#explorer_title_text")
 
-    btn_group_text = Input(name='name')
-    btn_group_hvr_text = Input(name='description')
-    btn_image = FonticonPicker('button_icon')
+    text = Input(name="name")
+    display = Checkbox(name="display")
+    hover = Input(name="description")
+    icon = FonticonPicker("button_icon")
+    icon_color = ColourInput(id="button_color")
 
 
 class ButtonForm(ServicesCatalogView):
-    title = Text('#explorer_title_text')
-
-    btn_text = Input(name='name')
-    btn_hvr_text = Input(name='description')
-    btn_image = BootstrapSelect('button_image')
-    select_dialog = BootstrapSelect('dialog_id')
-    system_process = BootstrapSelect('instance_name')
-    request = Input(name='object_request')
+    title = Text("#explorer_title_text")
 
     @View.nested
-    class options(WaitTab):    # noqa
-        TAB_NAME = 'Options'
-
-        btn_text = Input(name='name')
-        btn_hvr_text = Input(name='description')
-        select_dialog = BootstrapSelect('dialog_id')
-        btn_image = FonticonPicker('button_icon')
+    class options(WaitTab):  # noqa
+        btn_type = BootstrapSelect("button_type")
+        form = ConditionalSwitchableView(reference="type")
```

Comment:
Shouldn't this reference be 'btn_type', given that's the name of the `BootstrapSelect` widget above?

Reply:
It's my bad; you are not missing. I renamed `type` to `btn_type` to protect `type` default method and miss this. :) **The Eagle Eyes**